package.json

```diff
@@ -26,17 +29,23 @@
     "@types/reach__router": "^1.3.7",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.18.0",
+    "@typescript-eslint/parser": "^4.18.0",
+    "eslint": "^7.2.0",
```

Comment:
should we follow this as eslint is already part of the create-react-app scripts and the build process?
https://create-react-app.dev/docs/setting-up-your-editor#extending-or-replacing-the-default-eslint-config

Reply:
Extending the `react-app` and `react-app/jest` configurations might be a good idea.

`create-react-app` comes with eslint but we should also include it as dev dependency because `eslint-config-airbnb-typescript` has some peer dependencies and those have `eslint` as peer dependency. Things probably won't break without this dependency, but the warnings are annoying.